packages/import/src/index.ts

```diff
@@ -39,6 +39,7 @@ import { readFileSync, realpathSync } from 'fs';
 import { dirname, join, isAbsolute } from 'path';
 import resolveFrom from 'resolve-from';
 import { cwd as cwdFactory } from 'process';
+import { parseGraphQLSDL } from '@graphql-tools/utils';
```

Comment:
@patrickszmucer @ardatan This is importing package not declared by `package.json`. There might be cases when this fail to resolve (depending shape of `node_modules` tree or just in PnP mode). Any chance of adding some form of (peer)dependency?

Noticed this in test variant on our side that use PnP:
```
@graphql-tools/import tried to access @graphql-tools/utils, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: @graphql-tools/utils
Required by: @graphql-tools/import@virtual:d964d9031c12179c26af60039ca8671c524e04f097e5a562f4bb25437e4eb48d987bd211554a1caa331c24f4f529ba92cc0c246982fbd438d667cb1765c40962#npm:6.5.0 (via /tmp/e2e-tests/gatsby-pnp/.yarn/__virtual__/@graphql-tools-import-virtual-5780c4735f/0/cache/@graphql-tools-import-npm-6.5.0-f52472dd83-14532149f9.zip/node_modules/@graphql-tools/import/)
```

Reply:
Would you create a PR for this change?

Reply:
Apologies for this @pieh and thanks for spotting. I opened a PR to fix this here https://github.com/ardatan/graphql-tools/pull/3651

@ardatan maybe we can set up this lint rule https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md to catch things like this in the future?

Reply:
Good idea! Doing it now!